CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 ## Unreleased
 
+### Features
+
+- New Sentry KMP Gradle plugin ([#230](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/230))
+  - Install via `plugins { id("io.sentry.kotlin.multiplatform.gradle") version "{version}" }`
+  - Enables auto installing of the KMP SDK to `commonMain` (if all targets are supported)
+    - This also automatically installs the Sentry Android SDK
+  - Enables auto installing of the required Sentry Cocoa SDK with Cocoapods (if Cocoapods plugin is enabled)
+  - Configures linking for SPM (needed if you want to compile a dynamic framework with `isStatic = false`)
+  - Configure via the `sentryKmp` configuration block in your build file
+```kotlin
+// Example configuration in build.gradle.kts
+sentryKmp {
+  // Disable auto installing the KMP SDK to commonMain
+  autoInstall.commonMain.enabled = false
+} 
+```
+
+
 ### Dependencies
```

Comment:
`m`: if the changelog entry is supposed to replace the one for the beta (as implied by the updated KMP feature section) it should also repeat/update the dependencies that changed from the latest 0.7.x

Reply:
I added the Kotlin bump, the cocoa and android sdk bumps are already in the main branch